src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -106,6 +106,7 @@ private Cohort(Provider provider, AccountLegalEntity accountLegalEntity, Party o
         public virtual ICollection<TransferRequest> TransferRequests { get; set; }
 
         public IEnumerable<DraftApprenticeship> DraftApprenticeships => Apprenticeships.OfType<DraftApprenticeship>();
+        public bool IsApprovedByAllParties => EditStatus == EditStatus.Both && (TransferSenderId == null || TransferApprovalStatus == 1);
```

Comment:
Could we add an enum for TransferApprovalStatus and represent it this way instead of a nullable byte?

NULL | Not applicable/not sent to transfer sender
0 | Pending
1 | Approved
2 | Rejected